packages/api/src/graphql/query.private.ts

```diff
@@ -659,107 +659,107 @@ export const GraphQLQuery = new GraphQLObjectType<undefined, Context>({
         after = after ? JSON.parse(base64Decode(after)) : null
 
         const peers = await dbAdapter.peer.getPeers()
+        peers.filter(peer => (peerFilter ? peer.name === peerFilter : true))
```

Comment:
This does not work. `filter()` returns a new array with the filtered elements. But you can write it all on one line.

`const peers = await dbAdapter.peer.getPeers().filter(peer => (peerFilter ? peer.name === peerFilter : true))`